scss/boosted-grid.scss

```diff
@@ -3,9 +3,11 @@
 
 @import "functions";
 @import "variables";
+@import "variables-dark";
```

Comment:
Weird to implement dark mode inside grid 😄

Reply:
Yep it was mentioned in the color mode tracker and I've made a PR about it upstream.